### ISSUE TITLE
[Suggestion] Switch count badge and remove action

### ISSUE THREAD

**Issue by IllusionMH** [Microsoft/vscode-reference-view#13]
It would be easier to dismiss files and references if results Remove action buttons will be aligned.

At this moment if `explorer.decorations.badges` is `true` badge will be rendered for file entry and Remove button will be shifted. And shift won't be constant because badge width depends on errors/warnings and Git status.

**Comment by jrieken** [comment]
Yeah, that's kind of the design of the file label - decoration are the way to right, actions are left of that. Would be a bigger change. What will come for sure is a keybinding, like <kbd>Del</kbd>, to remove results
